include/seqan3/range/views/join.hpp

```diff
@@ -0,0 +1,26 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \brief Provides seqan3::views::join_with.
+ */
+
+#pragma once
+
+#include <range/v3/view/join.hpp>
+
+#include <seqan3/core/platform.hpp>
+#include <seqan3/std/ranges>
+
+namespace seqan3::views
+{
+
+//TODO reimplement me and rename to join_with
```

Comment:
Are there cards?
Might as well do \todo

Reply:
\todo is only parsed as part of doxygen comments..

Reply:
Why no do it as comment then.
Doxygen might complain anyway about it being not documented...

Reply:
Then I have to cond it to not make it appear in the user documentation, I really think having a card is the way to go and not use doxygen for this.